.github/workflows/tests.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python: [ 3.7, 3.9 ]
+        python: [ 3.7, "3.10" ]
```

Comment:
Maybe leave this on 3.9 for now, and update to 3.10 once msprime has got packages on 3.10?